bigquery/indexer.py

```diff
@@ -38,6 +38,13 @@ def parse_args():
         type=str,
         help='Directory containing config files. Can be relative or absolute.',
         default=os.environ.get('DATASET_CONFIG_DIR'))
+    parser.add_argument(
+        '--billing_project_id',
+        type=str,
+        help=
+        'The project to be billed for GCP operations. The account running this script must have bigquery.jobs.create on this project.',
```

Comment:
I prefer my wording. It makes it clear to the user what is being billed (so they have an idea of what they'll be charged).

This project will be billed for querying the BigQuery tables. The account running this script must have bigquery.jobs.create permission on this project.

Reply:
Done